sdk/servicebus/Azure.Messaging.ServiceBus/src/Amqp/AmqpSender.cs

```diff
@@ -457,18 +457,18 @@ public override async Task CloseAsync(CancellationToken cancellationToken)
                     }
 
                     entries.Add(entry);
+                    cancellationToken.ThrowIfCancellationRequested<TaskCanceledException>();
```

Comment:
This may be overkill; the intent was to save on the cost of allocations and serialization if cancellation occurred while processing messages.

Reply:
Yeah, I would vote to not include this.

Reply:
Sorry, missed this comment when closing out.   I'll throw a new PR out to remove it.

Reply:
#34537